gtsfm/evaluation/plot_metrics.py

```diff
@@ -30,21 +36,33 @@ def create_metrics_plots_html(json_path: str, output_dir: str) -> None:
         "rotation_averaging_metrics.json",
         "translation_averaging_metrics.json",
         "data_association_metrics.json",
-        "bundle_adjustment_metrics.json"
+        "bundle_adjustment_metrics.json",
     ]
+
+    if Path(colmap_files_dirpath).exists():
+        txt_metric_paths = {
+            "colmap": colmap_files_dirpath,
+        }
+        compare_metrics.compare_metrics(txt_metric_paths, json_path, GTSFM_MODULE_METRICS_FNAMES)
+    else:
+        logger.info("%s does not exist", colmap_files_dirpath)
+
+    metric_paths = []
     for filename in GTSFM_MODULE_METRICS_FNAMES:
         logger.info("Adding metrics from %s", filename)
         metric_path = os.path.join(json_path, filename)
+        metric_paths.append(metric_path)
         metrics_groups.append(GtsfmMetricsGroup.parse_from_json(metric_path))
     if len(output_dir) == 0:
         output_dir = json_path
     output_file = os.path.join(output_dir, "gtsfm_metrics_report.html")
-    metrics_report.generate_metrics_report_html(metrics_groups, output_file)
+    metrics_report.generate_metrics_report_html(metrics_groups, output_file, colmap_files_dirpath, metric_paths)
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--metrics_dir", default="result_metrics", help="Directory containing the metrics json files.")
+    parser.add_argument("--colmap_files_dirpath", default=None, type=str, help="Directory containing COLMAP output .")
     parser.add_argument("--output_dir", default="", help="Directory to save plots to. Same as metrics_dir by default.")
     args = parser.parse_args()
-    create_metrics_plots_html(args.metrics_dir, args.output_dir)
+    create_metrics_plots_html(args.metrics_dir, args.colmap_files_dirpath, args.output_dir)
```

Comment:
I think a better flow is like this: 

```
if args.colmap_files_dirpath is not None:
  save_colmap_metrics(args.colmap_files_dirpath, colmap_json_path)  # saves metrics to the json path
create_metrics_plots_html(args.metrics_dir, colmap_json_path, args.output_dir)
```

Reply:
basically separate metric computation from plotting.

Reply:
Thanks for the suggestion - the new design is better.